rs/bitcoin/kyt/src/logs.rs

```diff
@@ -0,0 +1,29 @@
+use ic_canister_log::declare_log_buffer;
+use serde::{Deserialize, Serialize};
+
+// High-priority messages.
+declare_log_buffer!(name = P0, capacity = 1000);
+
+// Low-priority info messages.
+declare_log_buffer!(name = P1, capacity = 1000);
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub enum Priority {
+    P0,
+    P1,
```

Comment:
Personally I would much prefer some standard `DEBUG`, `INFO` for example. I never know between `P0` or `P1` which one has the higher priority.

Reply:
Changed to WARN and DEBUG.